app/controller.js

```diff
@@ -5,8 +5,22 @@ novapostaApp.controller('novapostaController', function novapostaController($sco
         $scope.Description= $routeParams.Description;
 
     });
+    $scope.onMap = function (city){
+
+        novapostaService.onMap(function (data) {
+
+        }, city)
+    }
+
+
+
 });
 
+
+
+
```

Comment:
Why so many new lines? Could you separate your controllers into different files?